### DISCUSSION TITLE
Как сохранить настройки плагинов после замены версии far2l

### DISCUSSION THREAD

**Discussion by LanThrusteR** [elfmz/far2l#2334] [General]
настройки плагинов, в частности colorer находятся тут: 
/usr/share/far2l/Plugins/colorer
если их редактировать, то после обновления - настройки слетают, каким образом локализировать настройки плагина для пользователя, в частности colorer, так, чтобы после обновления версии far2l настройки не слетали? В какой форме и куда сохранить настройки /usr/share/far2l/Plugins/colorer в локальную папку пользователя чтобы их сохранить?

**Comment by unxed** [comment]
В ~/.config/far2l/plugins/colorer можно указывать свои настройки, которые буду применяться поверх базовых.
